CMakeLists.txt

```diff
@@ -46,6 +46,11 @@ if ((NOT TARGET c_build_tools) AND (EXISTS ${CMAKE_CURRENT_LIST_DIR}/deps/c-buil
     set_default_build_options()
 endif()
 
+if ((WIN32) AND ("${GBALLOC_LL_TYPE}" STREQUAL "JEMALLOC"))
+    # Bring in vcpkg
+    use_vcpkg(${CMAKE_CURRENT_LIST_DIR}/deps/vcpkg)
+endif()

```

Comment:
shouldn't this be done by c-pal subrepo?
or it be part of "canon" way of including c-pal?